schemars/tests/integration/snapshots/schemars/tests/integration/enums.rs~optionally_flattening_parent.json

```diff
@@ -0,0 +1,94 @@
+{
+  "$defs": {
+    "Struct": {
+      "properties": {
+        "bar": {
+          "type": "boolean"
+        },
+        "foo": {
+          "format": "int32",
+          "type": "integer"
+        }
+      },
+      "required": [
+        "foo",
+        "bar"
+      ],
+      "type": "object"
+    },
+    "UnitStruct": {
+      "type": "null"
+    }
+  },
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "anyOf": [
+    {
+      "anyOf": [
+        {
+          "type": "null"
+        },
+        {
```

Comment:
Something seems incorrect here.

This is:
```json
  "anyOf": [
    {
      "anyOf": [
        {
          "type": "null"
        },
        {
          ... enum ...
        }
      }
    },
    {}
  }
```

There should only be one `anyOf`. I don't think the check is specific enough.